skype/skype-ps/skype/New-CsTenantDialPlan.md

```diff
@@ -202,7 +201,7 @@ Accept wildcard characters: False
 
 ### -SimpleName
 The SimpleName parameter is a display name for the tenant dial plan.
-This name must be unique among all tenant dial plans within the Skype for Business Server deployment.
+This name must be unique among all tenant dial plans.
 
 This string can be up to 49 characters long.
 Valid characters are alphabetic or numeric characters, hyphen (-), dot (.) and parentheses (()).
```

Comment:
the rest if of the description of 'SimpleName' should probable be deleted or havily modified.

> This parameter must contain a value.
> However, if you don't provide a value, a default value will be supplied.
> The default value for a Global tenant dial plan is Prefix All.
> The default value for a site-level tenant dial plan is the name of the site.
> The default value for a service is the name of the service (Registrar or PSTN gateway) followed by an underscore, followed by the service fully qualified domain name (FQDN).
> For example: Registrar_pool0.litwareinc.com.
> The default value for a per-user tenant dial plan is the Identity of the tenant dial plan.

Reply:
Example 2 for New-CsTenantDialPlan is invalid.

The first line in the example should be:

```PowerShell
$nr2 = New-CsVoiceNormalizationRule -Identity 'Global/NR2' -Description 'TestNR1' -Pattern '^(\d{11})$' -Translation '+$1' -InMemory
```

instead of

```PowerShell
$nr2 = New-CsVoiceNormalizationRule -Identity Global/NR2 -Description "TestNR1" -Pattern '^(d{11})$' -Translation '+1' -InMemory
```